DEVELOPER.md

```diff
@@ -20,10 +20,11 @@ If you are new to Bazel take a look at the Workspace file, as it lays out which
 
 Just a quick recap of above.  Please install the following.
 
-1. Bazel
-1. Python - TODO version 
-1. Container Engine
-1. kubectl
+1. Bazel (do not install 4.0.x)
+2. realpath
+4. Python - TODO version 
+5. Container Engine
```

Comment:
You do not need docker for dev, unless you are doing open shift or pushing a release